tyk-headless/templates/secrets.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: secrets
+  namespace: {{ .Release.Namespace }}
```

Comment:
Explicitly specifying the `namespace` isn't needed — Helm will deploy all the namespaced resources into the same NS.

Reply:
Namespace is now removed.